docs/outline-style.md

```diff
@@ -45,40 +45,21 @@ Use `outline-none` to hide the default browser outline on focused elements.
 It is highly recommended to apply Warp:s focus styling for accessibility when using this utility.
 
 <container>
-<div class="flex flex-col sm:flex-row items-center justify-center gap-x-16 gap-y-8 px-0 sm:px-10 font-mono font-bold">
-<input type="text"
-  placeholder="Default focus style"
-  class="font-sans block pd-text-sm py-4 px-6 border-2 border-solid pd-border-slate-900/10 pd-text-slate-500 rounded-8 shadow-sm dark:pd-bg-slate-800 outline-inherit dark:pd-highlight-white/5 dark:pd-text-slate-400" 
-  style="outline: revert !important"
-  />
-
-<input type="text"
-  placeholder="No focus with outline-none"
-  class="outline-none rounded-8 font-sans block border-2 border-solid pd-border-slate-900/10 pd-text-sm w-full py-4 px-6 text-slate-500 rounded-lg shadow-sm" />
-
-<input type="text"
-  placeholder="Custom focus style"
-  class="focusable focus:[--w-outline-offset:-2px] rounded-8 font-sans block border-2 border-solid pd-border-slate-900/10 pd-text-sm w-full py-4 px-6 text-slate-500 rounded-lg shadow-sm" />
+  <div class="flex flex-col sm:flex-row items-center justify-center gap-x-16 gap-y-8 px-0 sm:px-10 font-mono font-bold">
+    <input type="text" class="focus:outline s-bg s-border rounded border border-solid py-4 px-6 w-full" placeholder="Default focus style" />
+    <input type="text" class="outline-none s-bg s-border rounded border border-solid py-4 px-6 w-full" placeholder="No focus with outline-none" />
+    <input type="text" class="focusable focus:[--w-outline-offset:-2px] s-bg s-border rounded border border-solid py-4 px-6 w-full" placeholder="Warp:s focus style" />
```

Comment:
What is `Warp:s` focus?

Reply:
Well.... I am not really sure... 🤷 😄 
I haven't changed the wording here... Just cleaned up the examples. Any suggestions? 🙂

Reply:
Maybe it was supposed to be `Warp's focus style` then 😉